packages/ckeditor5-link/src/ui/linkactionsview.ts

```diff
@@ -220,12 +248,18 @@ export default class LinkActionsView extends View {
 		button.bind( 'isEnabled' ).to( this, 'href', href => !!href );
 
 		button.template!.tag = 'a';
-		button.template!.eventListeners = {};
 
 		return button;
 	}
 }
 
+/**
+ * Checks if clicking the link can be scrolled to bookmark when bookmark `id` matches hash `url`.
+ */
+function isScrollableToBookmark( link: string | undefined, bookmarkEditing: BookmarkEditing | null ): boolean {
```

Comment:
Maybe this helper should be provided from the outside world as an options callback.